### ISSUE TITLE
17.07 missing openSSL

### ISSUE THREAD

**Issue by Robbilie** [processone/docker-ejabberd#3]
running the 17.07 image yields this error for me:

```
2017-08-19T14:27:58.670011871Z 2017-08-19 14:27:58 Unable to load crypto library. Failed with error:~n"~p, ~s"~nOpenSSL might not be installed on this system.~n
2017-08-19T14:27:58.670073212Z load_failed
2017-08-19T14:27:58.670081545Z "Failed to load NIF library: 'Error loading shared library libcrypto.so.38: No such file or directory (needed by /home/p1/ejabberd/lib/crypto-3.7.1/priv/lib/crypto.so)'"
2017-08-19T14:27:58.670145581Z 		2017-08-19 14:27:58 crash_report        
```

also, the latest image is the 16.12 tag, should probably be updated

**Comment by Robbilie** [comment]
i currently "fix" this by mounting a lot of stuff from the host system into the docker container.
i saw you updated the build file to 17.08, now that 17.09 is released you may want to update it again as well as push new images to docker hub and update the latest tag @cromain  :)

**Comment by cromain** [comment]
Yes, we'll push a working 17.09 container in October. First we need to update all hex.pm packages, then add all missing stuff and double check everything.

**Comment by cromain** [comment]
this has been fixed in 17.12 container
